node/src/IModelHubDownload.ts

```diff
@@ -3,12 +3,15 @@
 * See LICENSE.md in the project root for license terms and full copyright notice.
 *--------------------------------------------------------------------------------------------*/
 
-import { Logger } from "@bentley/bentleyjs-core";
-import { ElectronAuthorizationBackend } from "@bentley/electron-manager/lib/ElectronBackend";
-import { AuthorizedBackendRequestContext, BriefcaseDb, BriefcaseManager } from "@bentley/imodeljs-backend";
-import { LocalBriefcaseProps } from "@bentley/imodeljs-common";
-import { AccessToken } from "@bentley/itwin-client";
-import { APP_LOGGER_CATEGORY } from "./Main";
+import { Logger } from '@bentley/bentleyjs-core';
+import { ElectronAuthorizationBackend } from '@bentley/electron-manager/lib/ElectronBackend';
+import {
+    AuthorizedBackendRequestContext, BriefcaseDb, BriefcaseManager
+} from '@bentley/imodeljs-backend';

```

Comment:
don't we usually prefer `"` delimiters for our strings in JS? Minor nit, but seems like an unnecessary change here.

Reply:
bad settings from import orderer